### PR TITLE
doc: update generate_key instructions

### PR DESCRIPTION
**Pull request by dandanlen**

### CODE REVIEW

File: pages/perseverance/validator-setup/keys.mdx [hunk 1]
```diff
@@ -1,96 +1,75 @@
...
+**Make sure to back up your Seed Phrase and make a note of the public keys and account ID. You will need the Seed Phrase if you ever need to restore your node or recover your funds if you lose access to the node. DO NOT LOSE THIS.**
 
-<Callout type="error">**NEVER REVEAL YOUR PRIVATE KEYS TO ANYONE.**</Callout>
+*If the above command returned any errors, please ensure that the provided --path is accessible and that there are not already keys stored there! For security reasons, the files will not be overwritten automatically, so if you want to replace the stored keys, please make sure to move the old ones first, and back them up somewhere.*
 
-To finish off the work with the Signing Key, take a special note of the `Public Key (SS58)` of your Signing Key. **That it actually your Validator ID** that you will need to add funds and track your node.
+Take special note of the Validator Account ID beginning with `cF`. This is the ID that you will need to add funds and track your node.
 
-### Loading Your Signing Keys
+<Callout type="error">**NEVER REVEAL YOUR PRIVATE KEYS TO ANYONE.**<Callout>
```
Comment (acdibble) [inline pages/perseverance/validator-setup/keys.mdx:50]:
```suggestion
<Callout type="error">**NEVER REVEAL YOUR PRIVATE KEYS TO ANYONE.**</Callout>
```

**Review by fernandezdavid — APPROVED** [review]

File: pages/perseverance/validator-setup/keys.mdx (hunk 1 above)
Comment (dandanlen) [reply pages/perseverance/validator-setup/keys.mdx]: Thank you.

File: pages/perseverance/validator-setup/keys.mdx [hunk 2]
```diff
@@ -1,96 +1,75 @@
...
+```bash
+chainflip-cli generate-keys --path /etc/chainflip/keys
 ```
 
-**You must ensure that the public address administered by the private key at the above location has at least 0.1 gETH. Make sure you send 0.1 gETH to this account's address before trying to add funds.** This requirement is subject to change based on Goerli Ethereum transaction fees but for now should be sufficient.
+This should give you output similar to the below:
 
-## Validator Keys
+```log
+Generated fresh Validator keys for your Chainflip Node!
```
Comment (dandanlen) [inline pages/perseverance/validator-setup/keys.mdx:23]: I'm not sure how to make it preserver this whitespace inside the code block...

Comment (acdibble) [reply pages/perseverance/validator-setup/keys.mdx:23]: I'll have a look

Comment (dandanlen) [reply pages/perseverance/validator-setup/keys.mdx]: Of course! How silly of me! Thanks for this.

**Pull request merged**